packages/kbn-config-schema/src/byte_size_value/index.ts

```diff
@@ -35,9 +35,13 @@ export class ByteSizeValue {
   public static parse(text: string): ByteSizeValue {
     const match = /([1-9][0-9]*)(b|kb|mb|gb)/.exec(text);
     if (!match) {
-      throw new Error(
-        `could not parse byte size value [${text}]. Value must be a safe positive integer.`
-      );
+      const number = Number(text);
+      if (typeof number !== 'number' || isNaN(number)) {
+        throw new Error(
+          `could not parse byte size value [${text}]. Value must be a safe positive integer.`
+        );
```

Comment:
NIT: error message is (was already) false as it does not state the actual possible formats. `12kb` is valid, but not a 'safe positive integer'

Reply:
I'll change the error message to this:

`Failed to parse [${text}] as byte value. Value must be either number of bytes, or follow the format <count>[b|kb|mb|gb]. Number must also be a safe positive integer.`